.github/workflows/cwl_conformance.yml

```diff
@@ -0,0 +1,65 @@
+name: CWL Conformance Tests
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+    branches:
+      - main
+      - develop
+    types:
+      - opened
+      - synchronize
+      - ready_for_review
+      - labeled
+  workflow_dispatch:
+permissions:
+  contents: read
+  actions: write
+  packages: read
+jobs:
+  test:
+    name: CWL Conformance Tests (Staging)
+    if: contains(github.event.pull_request.labels.*.name, 'cwl_conformance')
+    runs-on: ubuntu-20.04
+    env:
+      PROJECT: dxCompiler_playground
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+      # performance optimizations https://abbbi.github.io/actions/
+      - uses: abbbi/github-actions-tune@v1
+      - name: Install java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Install dxpy and other dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y apt-transport-https wget git openssh-server tree python3 python3-pip python3-venv
+          cwltool=`jq -r '.execDepends.cwl[0].name' scripts/bundled_dependencies.json`
+          pip3 install --upgrade pip setuptools pyOpenSSL
+          pip3 install termcolor dxpy $cwltool
+          # install dxda
+          sudo wget -O/usr/local/bin/dx-download-agent https://github.com/dnanexus/dxda/releases/download/v0.5.7/dx-download-agent-linux
+          sudo chmod +x /usr/local/bin/dx-download-agent
+      - name: Run CWL conformance tests
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          ROBOT_TOKEN: ${{ secrets.DX_STAGING_ROBOT_TOKEN }}
+        run: |
+          export PATH="$PATH:$HOME/.local/bin"
+
+          # Set up DNAnexus staging environment
+          dx login --noprojects --staging --token $ROBOT_TOKEN
+          dx select $PROJECT
+          cd ${GITHUB_WORKSPACE}
+
+          # Run integration tests
+          FOLDER=/builds/test_gha_dxcompiler_$(date +%s)_$RANDOM
+          dx mkdir -p $FOLDER
+          python3 scripts/run_tests.py --folder $FOLDER --project $PROJECT \
+          --clean \
+          --test cwl_conformance_tools cwl_conformance_workflows
```

Comment:
```suggestion
          --test cwl_tools --test cwl_workflows
```

Reply:
See: https://github.com/dnanexus/dxCompiler/blob/3015b6bbc11c17938f4ec62789525143ade21ce9/scripts/run_tests.py#L551-L552
or we could change the names in the `run_tests.py`
@commandlinegirl